services/functions/graphql/out.mjs

```diff
@@ -0,0 +1,37 @@
+const DUMMY_RESOLVER = { serialize: x => x, parseValue: x => x }; 
```

Comment:
I'm curious why we need this `mjs` file

Reply:
`out.mjs` is not needed, and should not be included in the repos.  The file is an artifact of SST's "Live Lambda Development" environment feature.